bns/survey.js

```diff
@@ -129,8 +129,12 @@ upsert('WCSPROGRAMS_KoboBnsAnswer', 'AnswerId', {
   SubmissionUuid: dataValue('_uuid'),
   AnswerId: dataValue('_id'),
   LastUpdate: new Date().toISOString(),
-  SurveyDate: state =>
-    state.data.today ? state.data.today : state.data._submission_time,
+  SurveyDate: state => {
+    if (state.data.today.split('-')[0] >= 2014 ) {
+      return state.data.today ? state.data.today : state.data._submission_time
```

Comment:
if you reach this point in the code, that means `state.data.today` exists already so if you want to use either `today` or `_submission_time` I would suggest before the if you add something like: 
`const date = state.data.today || state.data._submission_time`
and on your if you will use date like `if(Number(date.split('-')[0]...)`